src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
```

Comment:
- [ ] Please, consider that you are not using this App.css style file and all styles inside accordingly. So, as it came from the CRA command, consider removing this style (in your case whole App.css as a file) 👍